AppDelegate.swift

```diff
@@ -39,6 +40,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, MessagingDelegate, UNUser
         )
 
         application.registerForRemoteNotifications()
+        
+        CLLocationManager().requestWhenInUseAuthorization()
```

Comment:
Can we create a method in DashXClient.swift in the SDK called requestLocationPermission() and call that instead